R/durations.r

```diff
@@ -348,6 +349,18 @@ setMethod("Compare", c(e1 = "ANY", e2 = "Duration"),
                  call. = FALSE)
           })
 
+#' @export
+setMethod("Compare", signature(e1 = "Duration", e2 = "numeric"),
+          function(e1, e2) {
+            callGeneric(e1, as.duration(e2))
```

Comment:
Sorry to still bug you on this, but I think as.duration is an unnecessary indirection here. You know that comparing to numeric means comparing seconds, so the numeric could be compared directly to e1@.Data as in the "character" method below. 

I even think that an ANY method with as.numeric could be an even better solutions here. But it might have some unexpected consequences though.

Reply:
No that's fine. Bug away. Let's get it right.  Which do you prefer then? I could try the ANY solution with some tests but I think you're right there may be unforeseen problems cause by that.

Reply:
For now, I've just changed the method to drop the unnecessary coercion and compare the `@.Data` instead. Makes much more sense.

Reply:
Thanks. Let's do this for now and I will think a bit more about ANY method. Most likely I won't implement it as it will create more room for bugs. For instance, you really don't want it to work on factors.